compatibility.bs

```diff
@@ -581,63 +582,11 @@ the corresponding unprefixed keyword:
 
 Issue(87): These definitions of <code>-webkit-box-*</code> are known to not be web compatible.
 
-<h4 id="the-webkit-background-clip-property">Foreground Text Clipping: the <code>
+<h4 id="the-webkit-background-clip-property"><code>
```

Comment:
Should we just stick this in the legacy name alias table? https://compat.spec.whatwg.org/#css-simple-aliases And then delete this whole section.